README.md

```diff
@@ -73,6 +73,62 @@ Now from the command line this is a lot nicer to use;
 bazel build //... --config clang-tidy
 ```
 
+### Use clang_tidy_test rule
+
+You cane use rule `clang_tidy_test` to test your files with clang-tidy.
+
+Example code is located here: `example/cc_test_example`
+
+You can run example with:
+
+```sh
+bazel test //example/cc_test_example:example_tests
+```
+
+This command is a `test_suite` for build and run your program and run clang-tidy on example files.
+
+To define clang-tidy test you simply add this rule to your `BUILD` file:
+
+```text
+load("@bazel_clang_tidy//clang_tidy:clang_tidy_test.bzl", "clang_tidy_test")
+clang_tidy_test(
+    name = '<TEST_NAME>',
+    srcs = [
+        "a.hpp"
+        "a.cpp"
+    ],
+)
+```
+
+In this rule, you can use the same arguments as in the aspect (they are public here - easier to set):
+
+```text
+'deps' : attr.label_list(),
+"cc_toolchain": attr.label(default = Label("@bazel_tools//tools/cpp:current_cc_toolchain")),
+"clang_tidy_wrapper": attr.label(default = Label("//clang_tidy:clang_tidy")),
+"clang_tidy_executable": attr.label(default = Label("//:clang_tidy_executable")),
+"clang_tidy_additional_deps": attr.label(default = Label("//:clang_tidy_additional_deps")),
+"clang_tidy_config": attr.label(default = Label("//:clang_tidy_config")),
+'srcs' : attr.label_list(allow_files = True),
+'hdrs' : attr.label_list(allow_files = True),
+'use_flags': attr.bool(default = True),
+```
+
+They can be set as follows:
+
+```text
+clang_tidy_test(
+  name = "clang_tidy_test",
+  clang_tidy_config = "//:clang_tidy_config",
+  clang_tidy_additional_deps = "//:clang_tidy_additional_deps",
+  srcs = [
+    ":test_sources"
+  ],
+  deps = ["some_deps"],
+  use_flags = False, # You want set this to false if you are using compile_commands.json
```

Comment:
i don't follow this, is this saying if you're using a compile_commands.json that's not managed by bazel you're expecting the test to pick that up instead?